### PR TITLE
Update servlet to 6.0.0 + other upgrades

### PR DESCRIPTION
**Pull request by wendigo**

### CODE REVIEW

**Comment by wendigo** [comment]
Trino built successfully with these changes.

**Comment by wendigo** [comment]
Cc @electrum

File: airbase/pom.xml [hunk 1]
```diff
@@ -422,13 +429,6 @@
...
             </dependency>
 
-            <!-- misc stuff -->
-            <dependency>
-                <groupId>org.eclipse.jetty.toolchain</groupId>
-                <artifactId>jetty-jakarta-servlet-api</artifactId>
-                <version>5.0.2</version>
-            </dependency>
-
             <!-- servlet -->
```
Comment (electrum) [inline airbase/pom.xml:433]: Remove this one too?

**Review by electrum — APPROVED** [review]

File: airbase/pom.xml (hunk 1 above)
Comment (wendigo) [reply airbase/pom.xml]: There are more javax.* dependencies to remove. I've added the removal in a separate commit.

**Pull request merged**